### ISSUE TITLE
new rule validate-comment-syntax

### ISSUE THREAD

**Issue by czjvic** [eslint/eslint#3651]
Source code
```javascript
///testing comment

/***
* testing comment
*/
```

Expected result
```
1:1 "///" is not valid start of inline comment
3:1 "/***" is not valid start of block comment
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by ilyavolodin** [comment]
I think the same behavior can be achieved by using [spaced-comment](http://eslint.org/docs/rules/spaced-comment). Not sure if a separate rule is required.

**Comment by gyandeeps** [comment]
@czjvic does the suggestion above by @ilyavolodin helps? If yes, then we close this issue but if no then can explain why?

**Comment by czjvic** [comment]
Hi,
I think I can not...

```javascript
//valid
// valid
///invalid
/// invalid
```

The rule force set space or not space after the "/" character.

**Comment by nzakas** [comment]
@czjvic we need a better description for what you are suggesting. We don't know what you want just from the code you provided.

**Comment by czjvic** [comment]
I want to validate syntax of comments. I want to force allow only two types of comment syntax.
1) inline - start with exactly 2 slashes 
```
//comment
// comment
```
- do not check space after slash
- check typo 
```
///comment
```
2) block - start with slash and 2 asterisk
```
/**
*comment
*/
- check typo
/***
*comment
*/
```

I think I can use "spaced-comment" when the space will be optional

```
eslint spaced-comment: [2, "optional"]
```
or
```
eslint spaced-comment: [2, "always", { "exceptions": ["a-zA-z"] }]
```

**Comment by nzakas** [comment]
@czjvic you would need to create a custom rule for your use case. `spaced-comment` only enforces spacing, it doesn't make sense to ignore it.
